doc/barclamp.md

```diff
@@ -1356,3 +1356,123 @@ place) you will have to add additional setup code to
 and/or
 [qa_crowbarsetup.sh](https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crobarsetup.sh)
 to take care of your barclamp's needs.
+
+## High Availability (HA) Integration
+
+[This commit](https://github.com/crowbar/crowbar-openstack/pull/512/files) has
+more or less everything that makes a barclamp HA compatible. So we can just base
+our understanding around that commit.
+
+-  [default.rb](https://github.com/crowbar/crowbar-openstack/pull/512/files#diff-f9a1e9d161032b6c0015336ab6812c14)
+
+   This file defines the default for all the values of the vars required by barclamp.
```

Comment:
This could be a bit more verbose. This file allows you to set defaults (or overrides) for any attributes in the node object. In this case we'll use that capability to set a bunch of HA related attributes that nobody will have any business changing, hence we won't need go to the trouble of putting it into the data bag.